.circleci/config.yml

```diff
@@ -628,4 +635,6 @@ workflows:
           requires:
             - build-linux
       - test-windows
-      - test-mac
+      - test-mac:
+          requires:
+            - build-linux
```

Comment:
Might be worth a comment here, "max build depends on linux build" seems surprising...